### DISCUSSION TITLE
Crazyflie with GPS

### DISCUSSION THREAD

**Discussion by scf-ku** [bitcraze/discussions#1549] [Q&A]
I am also trying to interfere a GPS module with crazyflie and uBlox MAX-7 GPS module refer to the following documents.(https://www.bitcraze.io/2014/03/crazyflie-with-gps-round-2)

![IMG_4049](https://github.com/user-attachments/assets/9cfa7673-2b75-4261-affc-2857c5d1514f)

This is a picture of crazyflie with GPS module.
I used Breakout deck and connected following way.
GPS module → Crazyflie
VCC → VCC
GND → GND
RxD → Tx2
TxD → Rx2

Next, I checked this issue(https://github.com/orgs/bitcraze/discussions/812) and enabled GpsTab.py and gpsTab.ui from tabs/__init__.py
But It didn't worked because PyQt6 does not support QtWebKit anymore.
I just would like to get GPS latitude and longitude data without GUI, Is there any sample program to get GPS data using python?

Assuming that, Is this connection method between crazyflie and GPS module correct?

**Comment by tobbeanton** [comment]
It is a very long time since we tried the [GPS code](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/deck/drivers/src/gtgps.c) and there is no official support for it. I don't even think the gtgps.c file is getting compiled and looking in that file it is uar1 functions that is being used.
